modules/ROOT/pages/compatibility-issues-runtime-java.adoc

```diff
@@ -0,0 +1,15 @@
+= Troubleshooting Issues with Mule Runtime and Java
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The modules in your project can have compatibility issues with Java or Mule runtime.
+
+Mule runtime 4.6 is compatible with Java 17, and Mule runtime 4.5 and earlier are compatible with Java 8 and 11.
```

Comment:
> Mule runtime 4.6 is compatible with Java 17
I would clarify that in addition to 8 and 11

Reply:
@mdaloia, please, can you elaborate on your comment?
The sentence above clarifies that Mule runtime 4.6 is compatible with Java 17.
Apart from that, it clarifies that Mule runtime 4.5 and earlier is compatible with Java 8 and 11.
Did I miss something?
Thanks!

Reply:
I interprete that sentence as that Runtime 4.6 is ONLY compatible with Java 17... and that is not true... 4.6 is compatible with 8, 11 AND 17.
Maybe is just my interpretation.

Reply:
Great that you mentioned it! I fixed the sentence to avoid misunderstandings. Thanks!